app/main.py

```
@@ -123,6 +131,32 @@ async def lifespan(_app: FastAPI):
+            )
+    except Exception as e:
+        logger.error(f"Failed to start Redis refresh worker: {e}")
+
+    # Initialize Feature Flags from DevCycle
+    try:
+        feature_store = get_feature_store()
+        await feature_store.initialize_from_devcycle()
+        logger.info(
+            f"Feature flags initialized: {feature_store.get_flag_count()} flags loaded"
+        )
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)